social-network/src/components/MissedCard.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+function MissedCard(props) {
```

Comment:
Note that this is a good candidate to combine with the other article component, since this logic is more-or-less identical.